src/commands/notify.yml

```diff
@@ -64,6 +75,9 @@ steps:
         run () {
           verify_api_key
           parse_jira_key_array
+
+          echo "JIRA_KEYS=${ISSUE_KEYS}"
```

Comment:
Were you meant to use JIRA_KEYS something?

Reply:
Haha no, that was me trying to work out why it was saying it hadn't picked up any keys